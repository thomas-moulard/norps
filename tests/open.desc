Teste les fonctions open, libs et close.
