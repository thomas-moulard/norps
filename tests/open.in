open libc.so as t
open libc.so as t2
open libz.so as t2
libs
close t2
close t2
close t