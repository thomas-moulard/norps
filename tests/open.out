~ libc is opened as t
~ libz is opened as t2
